source/extensions/stat_sinks/common/statsd/statsd.cc

```diff
@@ -67,7 +67,12 @@ void UdpStatsdSink::flush(Stats::MetricSnapshot& snapshot) {
 }
 
 void UdpStatsdSink::onHistogramComplete(const Stats::Histogram& histogram, uint64_t value) {
-  // For statsd histograms are all timers.
+  // TODO(#8594): for statsd histograms are all timers in milliseconds, Envoy histograms are however
```

Comment:
Personally, I would downgrade from TODO and just leave a note here in case this comes up again. Tons of people use Envoy with statsd and I have never heard anyone mention this problem with our stats histogram implementation. If/when someone actually cares about this we can revisit. Same below.

Reply:
Removed the TODO and left as a note. There are not many of these right now and they're all in extensions so I suspect the users would not complain but it's something to consider going forward.